docs/framework/tools/developer-command-prompt-for-vs.md

```diff
@@ -1,93 +1,135 @@
+Usually, the shortcuts for the shells you have installed are placed in the **Start Menu** folder for Visual Studio, such as in *%ProgramData%\Microsoft\Windows\Start Menu\Programs\Visual Studio 2019\Visual Studio Tools*. But if searching for the command prompt doesn't produce the expected results, you can try to manually locate the files on your machine.
 
-## Start the command prompt from inside Visual Studio
+#### Developer Command Prompt
 
-For easier access, you can add Developer Command Prompt, or any other command prompt, to the Tools menu in Visual Studio. To make the tool available, add it to the external tools list. Here are the steps:
+Search for the name of the command prompt file, which is *VsDevCmd.bat*, or go to the Tools folder for Visual Studio, such as *%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community\Common7\Tools* (path changes according to your Visual Studio version, edition, and installation location).
 
-1. Open Visual Studio.
+Once you've located the command prompt file, open it by entering the following command in a regular command prompt window:
 
-1. On the start window, choose **Continue without code**.
+```cmd
+"%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community\Common7\Tools\VsDevCmd.bat"
+```
 
-1. On the menu bar, choose **Tools** > **External Tools**.
+Or enter the following command in the Windows **Run** dialog box:
 
-1. On the **External Tools** dialog box, choose the **Add** button. A new entry appears.
+```cmd
+%comspec% /k "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\Tools\VsDevCmd.bat"
+```
 
-1. Enter a **Title** for your new menu item such as `Command Prompt`.
+> [!TIP]
+> You'll need to edit the path to match your Visual Studio installation.
 
-1. In the **Command** field, specify the file you want to launch, such as `%comspec%` or `C:\Windows\System32\cmd.exe`.
+#### Developer PowerShell
 
-1. In the **Arguments** field, specify where to find the specific command prompt you want to use, such as `/k "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\Tools\VsDevCmd.bat"`. This command launches the Developer Command Prompt that's installed with Visual Studio 2019 Community. Change this value according to your Visual Studio version, edition, and installation location.
+Search for a file named *Microsoft.VisualStudio.DevShell.dll*, or go to the Tools folder for Visual Studio, such as *%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community\Common7\Tools* (path changes according to your Visual Studio version, edition, and installation location).
```

Comment:
We should probably point the user to this PowerShell script instead (or in addition to ): & 'C:\Program Files (x86)\Microsoft Visual Studio\2019\int.main\Common7\Tools\Launch-VsDevShell.ps1'

1. Should be started from a  Windows PowerShell or PowerShell 6 prompt
2. ExecutionPolicy will need to be set to allow it to run
3. By default, requires no arguments like InstanceId. Defaults to being configured for the VS whose install path it is located in